### PR TITLE
T/matt 1833 autofill admin ui upload

### PR DESCRIPTION
**Pull request by spiraleddy**

### CODE REVIEW

**Comment by spiraleddy** [comment]
I can't see where the conflicts are....

**Comment by djcp** [comment]
I think you issued your PR right after I had merged my stuff onto develop, so I think it's in the README. Can you rebase this against develop and force push your fix to this branch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.4 - 1/15/2016
```
Comment (djcp) [inline CHANGELOG.md]: This should go into "To be released", the release manager will actually come up with the tag version when we release the new chef recipes.

File: templates/default/mh_default_org.cfg.erb [hunk 2]
```diff
@@ -116,12 +116,22 @@ prop.adminui.i18n_tab_users.enable=false
...
+# Note: this populates episodes dublincore with series dublincore. It uses series as a template.
 #
 # Values: true | false
 # Default: false
 #
 prop.adminui.series_prepopulate.enable=false
 
+# DCE MATT-1833, pre-populate dce_episode (using Otherpubs service endpoint)
+# Note: this populates episodes dublincore with episode default data. This allows
+# the series to have different, offering specific, dublincore data from it's episodes.
```
Comment (djcp) [inline templates/default/mh_default_org.cfg.erb]: *its

**Comment by spiraleddy** [comment]
I've rebased against current develop, fixed the typo seen by DJCP, and it's been deployed to a testing server and passed QA. The companion MH code has already been merged to MH repo develop branch and ready for this Friday's release along with this change.

**Pull request merged**